Sources/ArcGISToolkit/Components/FeatureFormView/FeatureFormView.swift

```diff
@@ -307,6 +308,17 @@ public extension FeatureFormView {
         return copy
     }
     
+    /// Sets a closure to perform when a new feature form is shown in the view.
+    ///
+    /// This can happen when navigating through the associations in a `UtilityAssociationsFormElement`.
+    /// - Parameter action: The closure to perform when the new feature form is shown.
+    /// - Since: 200.8
+    func onFeatureFormChanged(perform action: @escaping (FeatureForm) -> Void) -> Self {
```

Comment:
Isn't the SwiftUI View modifier `onChange(perform:)`, if so, then this should be `onFeatureFormChange(perform:)`?

Reply:
onPopupChanged should probably change as well